README.md

```diff
@@ -95,10 +95,39 @@ The list below show all build time dependencies in alphabetical order. Note that
 | [Apache Maven](https://maven.apache.org/)                                        | Build tool                                             | Apache License 2.0            |
 | [License Maven Plugin](https://www.mojohaus.org/license-maven-plugin/)           | Add licenses to source files automatically             | GNU Public License 3.0        |
 | [Maven Compiler Plugin](https://maven.apache.org/plugins/maven-compiler-plugin/) | Maven provided and controlled Java compiler            | Apache License 2.0            |
-
+| [Maven Source Plugin](https://maven.apache.org/plugins/maven-source-plugin/)     | Create Source JAR packages                             | Apache License 2.0            |
+| [Maven JavaDoc Plugin](https://maven.apache.org/plugins/maven-javadoc-plugin/)   | Create JavaDoc JAR packages                            | Apache License 2.0            |
 
 ### Essential Build Steps
 
 * `git clone https://github.com/itsallcode/junit5-system-extensions.git`
 * Run `mvn test` to run unit tests.
 * Run `mvn package` to create the JAR file.
+
+### Publishing to JCenter
+
+1. Add the following to your `~/.m2/settings.xml`:
+
+    ```xml
+    <servers>
+        <server>
+            <id>itsallcode-maven-repo</id>
+            <username>[bintray-username]</username>
+            <password>[bintray-api-key]</password>
+        </server>
+    </servers>
+    ```
+
+1. Checkout the `develop` branch.
+1. Update version in `pom.xml` and `README.md`, commit and push.
+1. Run command
+
+    ```bash
+    mvn deploy
+    ```
+1. Merge to `master` branch
+1. Create a [release](https://github.com/itsallcode/junit5-system-extensions/releases) of the `master` branch on GitHub.
+1. Sign in at [bintray.com](https://bintray.com)
+1. Go to the [Bintray project page](https://bintray.com/itsallcode/itsallcode/junit5-system-extensions)
+1. There should be a notice saying "You have 6 unpublished item(s) for this package". Click the "Publish" link. Binaries will be available for download at [JCenter](https://jcenter.bintray.com/org/itsallcode/junit5-system-extensions/)
```

Comment:
```suggestion
```

We have `publish=1` in the repo url in `pom.xml`, so artifacts are published automatically.